common/src/main/scala/com/aliyun/emr/rss/common/util/ThreadUtils.scala

```diff
@@ -106,6 +107,13 @@ object ThreadUtils {
     executor
   }
 
+  def newDaemonSingleThreadExecutorWithUncaughtExceptionHandler(
+      threadName: String,handler: UncaughtExceptionHandler): ExecutorService = {
+    val threadFactory = new ThreadFactoryBuilder().setDaemon(true).setNameFormat(
+      threadName).setUncaughtExceptionHandler(handler).build()
+    Executors.newSingleThreadExecutor(threadFactory)
+  }
+
```

Comment:
`newDaemonSingleThreadScheduledExecutor()` wrap with a ScheduledThreadPoolExecutor,  `newDaemonSingleThreadExecutorWithUncaughtExceptionHandler()` not do this, also can we merge this method together? since it's a scala class, we can used default value `handler=null`
```

  /**
   * Wrapper over ScheduledThreadPoolExecutor.
   */
  def newDaemonSingleThreadScheduledExecutor(threadName: String): ScheduledExecutorService = {
    val threadFactory = new ThreadFactoryBuilder().setDaemon(true).setNameFormat(threadName).build()
    val executor = new ScheduledThreadPoolExecutor(1, threadFactory)
    // By default, a cancelled task is not automatically removed from the work queue until its delay
    // elapses. We have to enable it manually.
    executor.setRemoveOnCancelPolicy(true)
    executor
  }

  def newDaemonSingleThreadExecutorWithUncaughtExceptionHandler(
      threadName: String,handler: UncaughtExceptionHandler): ExecutorService = {
    val threadFactory = new ThreadFactoryBuilder().setDaemon(true).setNameFormat(
      threadName).setUncaughtExceptionHandler(handler).build()
    Executors.newSingleThreadExecutor(threadFactory)
  }
```